### ISSUE TITLE
Not working since upgrade to 13.0.3

### ISSUE THREAD

**Issue by dazza120** [BottlecapDave/HomeAssistant-OctopusEnergy#1050]
### Describe the bug

Its been working grate for a few upgrades that ive done with the Octopus intergration, the api key s still the same and my account number is still in the intergration.

### Reproduction steps

Does not start just says needs attention

### Expected behaviour

Does not start just says needs attention

### Tariff Code

unable to obtatain as app does not start

### Integration Version

13.0.3

### Home Assistant Version

 Core 2024.10.4 Supervisor 2024.10.3 Operating System 13.2 Frontend 20241002.4

### Fresh Install?

After upgrading

### Home Assistant Logs

The intergration does not start so no logs from the meters are obtainable

### Confirmation

- [X] I confirm that I cannot find my solution within the [documentation](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy)
- [X] I confirm that I cannot find my solution within the [FAQ](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/faq)

**Comment by ianp574** [comment]
Seeing the same issue, upgraded to 13.0.3, integration showing as requires attention and no data populating.

**Comment by andymk3** [comment]
Same issue here. Downgraded to previous release and working properly again.

**Comment by BottlecapDave** [comment]
Sorry you're seeing this issue. Does anyone have any related logs as I'm not seeing this issue on either of my instances? The OE APIs have been a bit flakey over the last couple of days, which the integration requires on startup to get your up-to-date meter information.

**Comment by ianp574** [comment]
Sorry no, I downgraded to previous version (restored backup) and the integration was then working as expected, just upgraded again to the 13.0.3 version and the issue is not now present.

**Comment by BottlecapDave** [comment]
Sorry @ianp574 are you saying it's now working on v13.0.3 for you?

**Comment by ianp574** [comment]
> Sorry @ianp574 are you saying it's now working on v13.0.3 for you?

Yes, all working fine here today on 13.0.3, so it may just have been a timing issue with the api yesterday after I did my original update. I did leave it a few hours after the problem occurred and went through a few reauthorisation cycles with no success at that time. I then restored the previous version via a backup which started pulling data instantly.
This morning I simply applied the update to 13.0.3 to the restored backup and it was all OK.

**Comment by dazza120** [comment]
@BottlecapDave i just redownloaded the same version and it’s now working 13.0.3

**Comment by BottlecapDave** [comment]
Thanks for getting back to me. Is this the same for you @andymk3?

It sounds like there was some issues on OE side, which I've noticed over the past couple of days. I think they've been having some daylight saving issues. 

v13.0.3 didn't really change much (assuming you've upgraded from v13.0.2) other than add some additional logging and updates to the diagnostics functionality to include some additional information. I plan to try and have the integration fallback to some cached data if the data isn't available upon start up, as this comes up from time to time.

**Comment by David-DBSS** [comment]
I have also been seeing problems with the Octopus integration over the last few days. I had accidentally upgraded from 13.0.2 to 3 around the same day as the BST to GMT time change and my Energy dashboard energy prices have been zero since then. I did a little investigating this evening and saw the Needs Attention warnings. There was no clear fault to correct, but I came upon this thread. I did a redownload of 13.0.3 and have started seeing Logbook events. The numbers in the Energy dashboard are still zero, but maybe they will sort themselves out...
